src/chains/views.py

```diff
@@ -14,7 +14,7 @@
 class ChainsListView(ListAPIView):
     serializer_class = ChainSerializer
     pagination_class = LimitOffsetPagination
-    pagination_class.max_limit = 100
+    pagination_class.max_limit = 20
```

Comment:
Isn't this changing the class attributes for every view using it?

Reply:
So I guess every view that has the pagination class instantiated after this will have `max_limit=20`

Reply:
Yes you are right. I think it's better to remove this for now and add it to the Django configuration side 🤦

Reply:
I wouldn't do that there. I would define a custom LimitOffsetPagination and set the values there

Reply:
Done a72f7f1